style.css

```diff
@@ -0,0 +1,65 @@
+@media only screen and (max-width: 600px) {
+  .content {
+    background-image: url("image1.png");
+  }
```

Comment:
- [ ] Please make sure you use `flexbox` to align your content as you would be using it a lot as a developer and it is required for this project. You can read more about `flexbox` in this [article](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) 🙏 .